File: 101-APCS/Contents/c294/c294.c
#include<stdio.h>
#include<stdlib.h>

int cmp(const void *, const void *);
void whichTri(int, int, int);

int main(void)
{
	int sides[3];
	
	while(scanf("%d %d %d", &sides[0], &sides[1], &sides[2]) != EOF)
	{
		qsort(sides, 3, sizeof(int), cmp);
		
		printf("%d %d %d\n", sides[0], sides[1], sides[2]);
		
		whichTri(sides[0], sides[1], sides[2]);
	}
	
	return 0;
}

int cmp(const void *a, const void *b)
{
    return *(const int*)a - *(const int*)b;
}

void whichTri(int a, int b, int c)
{
	if(a + b <= c)
		printf("No\n");
	else if(a * a + b * b == c * c)
		printf("Right\n");
	else if(a * a + b * b < c * c)
		printf("Obtuse\n");
	else
		printf("Acute\n");
}

File: 101-APCS/Contents/c294/c294.py
import sys

for temp in sys.stdin:
    side = map(int, temp.split())

    side = sorted(side)
    print(*side)

    if side[0] + side[1] <= side[2]:
        print("No")
    elif side[0] ** 2 + side[1] ** 2 == side[2] ** 2:
        print("Right")
    elif side[0] ** 2 + side[1] ** 2 < side[2] ** 2:
        print("Obtuse")
    else:
        print("Acute")
